contribs/emissions/src/main/java/org/matsim/contrib/emissions/utils/EmissionUtils.java

```diff
@@ -44,6 +48,8 @@
 public class EmissionUtils {
 	private static final Logger logger = Logger.getLogger(EmissionUtils.class);
 
+	@Deprecated // yyyy we don't like static fields, especially not with mutable material.  Pls explain why this one here is
```

Comment:
Fine with me.  I would still like to merge this here since it is mostly (except for disallowing certain xml config options) a collection of design comments.

Reply:
There is some design decision that we will not silently ignore config params.  
We have in the past lived with breaking config params since versioning the config file seems too much effort (too many small changes).  We usually provide code that provides help (by throwing exceptions in the old setters/getters rather than removing them).